tests/automation_framework/src/utilities/submit_request_utility.py

```diff
@@ -64,16 +64,41 @@ def submit_request_listener(
         uri_client, input_json_str1, output_json_file_name):
     logger.info("Listener code path\n")
     req_time = time.strftime("%Y%m%d_%H%M%S")
+    request_method = input_json_str1["method"]
```

Comment:
Rename input_json_str1 to just input_json_str in this statement and the function definition.

Reply:
done